File: python/algorithm/everything_algorithm/algorithm/buble-sort/bubble-sort.py
arr = [5, 3, 8, 1]

for i in range(len(arr) - 1):
    for j in range(0, len(arr) - i - 1):
        if arr[j] > arr[j + 1]:
            arr[j], arr[j + 1] = arr[j + 1], arr[j]

File: JavaScript/algorithm/src/algorithm/bubble-sort/bubbleSort.js
function bubbleSort(arr) {
    let len = arr.length;
    for (let i = 0; i < len; i++) {
        for (let j = 0; j < len - 1 - i; j++) {
            if (arr[j] > arr[j + 1]) {
                let temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = temp;
            }
        }
    }
    return arr;
}

let myArray = [64, 34, 25, 12, 22, 11, 90];
console.log("Sorted array: ", bubbleSort(myArray)); // 결과: [11, 12, 22, 25, 34, 64, 90]